packages/swing-store/src/internal.js

```diff
@@ -8,7 +10,13 @@
  *    snapStore: SnapStoreInternal,
  *    bundleStore: BundleStoreInternal,
  * }} SwingStoreInternal
+ *
+ * @typedef {'operational' | 'replay' | 'archival' | 'debug'} ArtifactMode
  */
 
-// Ensure this is a module.
-export {};
+export const artifactModes = ['operational', 'replay', 'archival', 'debug'];
+export function validateArtifactMode(artifactMode) {
+  if (artifactModes.indexOf(artifactMode) === -1) {
+    Fail`invalid artifactMode ${q(artifactMode)}`;
+  }
```

Comment:
```suggestion
  artifactModes.includes(artifactMode) || Fail`invalid artifactMode ${q(artifactMode)}`;
```

Reply:
Thanks.. I'll switch to `includes`, which is awesome and I keep forgetting it's available, but I'm going to stick with the if/then rather than the short-circuit boolean, which I don't like. (I will note that I dislike it less in this case, where the conditional is really simple and reads like english, than in cases with `!` and `&&` and other combiners, where it takes extra mental energy to visually parse and understand). "if not good thing then fail" vs "good thing or fail".